programming/sr/cheat_sheet.md

```diff
@@ -195,28 +195,28 @@ for marker in markers:
 ### Setting the mode of a pin
 
 ~~~~~ python
-R.ruggeduinos[0].pins[4].mode = OUTPUT
-R.ruggeduinos[0].pins[4].mode = INPUT
-R.ruggeduinos[0].pins[4].mode = INPUT_PULLUP
+R.ruggeduinos['1234567890'].pins[4].mode = OUTPUT
```

Comment:
This should `R.ruggeduino` (singular), as it can be difficult to find the serial number, and there should only be 1 anyway.